extension/ts/main/extensions/util/onLoggedIn.ts

```diff
@@ -0,0 +1,32 @@
+import {invokeWorker} from "../../../common/workers/invoker";
+import {WorkerKind} from "../../../common/workers/types";
+import {createIconButton} from "../../ui/button";
+import {isAuthorized} from "../author/util/isAuthorized";
+
+const LOGIN_BUTTON_ID = "vbl-bll-login";
```

Comment:
I'm not sure I am a fan of this button having a specific ID.

The problem is then we can have at most one button per page. I think you could get around this but capturing the button in the scope of `removeInjectedButton`, could be defined in the scope of `onLoggedIn`